shell/app/modules/project/common/components/issue/edit-issue-drawer.tsx

```diff
@@ -128,6 +128,29 @@ const IssueMetaFields = ({ labels, isEditMode, isBacklog, editAuth, issueType, f
     ));
   }, [taskTypeList]);
 
+  React.useEffect(()=>{
+    if(ref && !ref.current){
+      const customFieldKeys = map(customFieldDetail?.property || [], 'propertyName').concat(['taskType', 'bugStage', 'owner']);
+      ref.current = {
+        onFocus: (fKey: string) => {
+          const curRef = ref.current?.refMap[fKey];
+          if(fKey === 'issueManHour.elapsedTime'){
+            // 点击打开时间跟踪
+            curRef.click();
+          } else if(customFieldKeys.includes(fKey)){
+            //需要打开自定义
+            setExpandCustomFields(true)
+            setTimeout(()=>{
+              curRef?.focus && curRef.focus();
```

Comment:
`curRef?.focus()` directly

Reply:
could get a error when curRef is exist and focus does't exist;